geometry/r3_element_body.hpp

```diff
@@ -68,7 +69,7 @@ inline Scalar& R3Element<Scalar>::operator[](int const index) {
     case 2:
       return z;
     default:
-      LOG(FATAL) << __FUNCSIG__ << ": index = " << index;
```

Comment:
Removing the `__FUNCSIG__` bothers me since it's going to be messy to put them back.  @eggrobin or myself will prepare a PR to add a portable macro to replace `__FUNCSIG__` and we'll come back to you.